tests/roles/manila_adoption/tasks/netapp.yaml

```diff
@@ -8,11 +8,26 @@
         CONTROLLER1_SCP="{{ controller1_ssh  | regex_replace('^ssh', 'scp')}}"
         ${CONTROLLER1_SCP}:{{ manila_tripleo_path }} {{ manila_conf_path }}
 
-    - name: Extract Netapp data from manila.conf
-      ansible.builtin.set_fact:
-        manila_netapp_config: "{{ manila_netapp_config | default({}) | combine({item: lookup('ansible.builtin.ini', item, file=manila_conf_path, section=manila_netapp_backend, allow_no_value=True)}) }}"
-      loop: "{{ manila_netapp_vars }}"
+    - name: Stat the retrieved manila.conf file
+      ansible.builtin.stat:
+        path: "{{ manila_conf_path }}"
+      register: manila_conf
+
+    - name: Fail if manila.conf is not present
+      when: not manila_conf.exists
```

Comment:
this should be manila_conf.stat.exists

Otherwise:
```
TASK [manila_adoption : Fail if manila.conf is not present] ********************
fatal: [localhost]: FAILED! => {"msg": "The conditional check 'not manila_conf.exists' failed. The error was: error while evaluating conditional (not manila_conf.exists): 'dict object' has no attribute 'exists'. 'dict object' has no attribute 'exists'\n\nThe error appears to be in '/home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption/tests/roles/manila_adoption/tasks/netapp.yaml': line 16, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Fail if manila.conf is not present\n      ^ here\n"}
```

Reply:
++ thank you for fixing it :D